.github/workflows/test.yml

```diff
@@ -17,6 +17,8 @@ jobs:
       uses: codespell-project/actions-codespell@master
       with:
         check_filenames: true
+        ignore_words_list: UES
+        uri_ignore_words_list: UES
```

Comment:
```suggestion
        ignore_words_list: UES,ues
        uri_ignore_words_list: UES,ues
```

Reply:
So we'll test this, based on https://github.com/easybuilders/easybuild-tutorial/pull/131#issuecomment-1408365050 - if this works we should add an explanation as well.

Reply:
Does indeed work, let's add a comment above to make it clear why the `ues` has to be there too?

Reply:
done in 94fdc2f